elvish-mode.el

```diff
@@ -63,7 +63,7 @@ expected.")
 
 (defconst elvish-keyword-pattern
   (let ((keywords (cons 'or elvish-keywords)))
-    (eval `(rx (group ,keywords) (not word))))
+    (eval `(rx (not (any word ?_ ?-)) (group ,keywords) (not (any word ?_ ?-)))))
```

Comment:
Thanks for the feedback! I also thought of defining the delimiter as a constant, will take a stab at it and also at fixing the "leading delimiter" problem.